qcengine/compute.py

```diff
@@ -3,7 +3,8 @@
 """
 from typing import Any, Dict, Optional, Union
 
-from qcelemental.models import AtomicInput, FailedOperation
+from pydantic.main import BaseModel
+from qcelemental.models import AtomicInput, FailedOperation, AtomicResult
```

Comment:
I can remove these (`BaseModel` and `AtomicResult`), I just added them in since PyCharm noticed they weren't defined when they are used to type declare the return type of `compute_procedure` and `compute` respectively

Reply:
Use the `if TYPE_CHECKING` pattern:
https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles